rpm/update-spec-version.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# This script will update the Version field in the spec which is set to 0 by
+# default. Useful for local manual rpm builds where the Version needs to be set
+# correctly.
+
+SPEC_FILE=$(pwd)/podman.spec
```

Comment:
Is this really `$(pwd)`? I looked in `.packit.yml` and don't see where it cd's to the `rpm` directory?

I would rather see something like `$(dirname $0)/podman.spec`, it seems safer. But it's very likely that I'm missing a key piece of Packit magic.

Reply:
`update-spec-version.sh` is only if someone needs a manual build or use `make package` locally. Doesn't get used by packit. I can update the comment to make that clear.